.chloggen/changes.yaml

```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
```

Comment:
since the metric is turned on by default, it should be a breaking change.

Reply:
No longer default :)